pkg/shardddl/optimism/info.go

```diff
@@ -84,6 +102,40 @@ func (i Info) String() string {
 	return s
 }
 
+// ShortString returns short string of Info
+func (i *Info) ShortString() string {
```

Comment:
I mean we could use value receiver, to avoid clone it in this function.

https://play.golang.org/p/qBG5ZRSFK0y

Reply:
But the struct between Info and LogInfo is different, unless we define another marshall method for info.